HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -27,11 +29,14 @@ HLTBTagHarvestingAnalyzer::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
 	{
 		dqmFolder_hist = Form("HLT/BTag/Discriminator/%s",hltPathNames_[ind].c_str());
 		std::string effDir = Form("HLT/BTag/Discriminator/%s/efficiency",hltPathNames_[ind].c_str());
+		std::string relationsDir = Form("HLT/BTag/Discriminator/%s/HEP17_HEM17",hltPathNames_[ind].c_str());
 		ibooker.setCurrentFolder(effDir);
 		TH1 *den =NULL;
 		TH1 *num =NULL; 
 		std::map<TString,TH1F> effics;
 		std::map<TString,bool> efficsOK;
+		std::map<TString,std::map<std::string,TH1F> > efficsmod;
```

Comment:
@fwyzard well yes, it is better to use std::string instead of TString, but it is out of scope of the current PR. The purpose of this one is not to improve the current code, but just to add the necessary HEP17 plots.
To my mind it's better to separate these two tasks.